templates/nginx.conf.j2

```diff
@@ -31,7 +31,14 @@ http {
 
     log_format  main  {{ nginx_log_format|indent(23) }};
 
+    {% for format_name, format in nginx_log_formats.iteritems() -%}
```

Comment:
I guess this should be a "nginx_log_formats.items", or?

Reply:
huh, it worked on my system. Looks like it's a python 2 vs python 3 issue. I'll rework it.